scaffoldmaker/utils/tubemesh.py

```diff
@@ -20,13 +20,14 @@ def generatetubemesh(region,
     elementsCountThroughWall,
     segmentCountAlong,
     cx, cd1, cd2, cd12,
-    xInner, d1Inner, d2Inner, wallThickness,
-    segmentAxis, segmentLength,
+    innerRadiusList, dInnerRadiusList,
+    tcWidthList, dTCWidthList,
```

Comment:
Can't pass colon-specific parameters in this way as it takes away the generic-ness of tubemesh.
All varying properties need to be passed to the segment generator, including radius and TC width. This makes sense as they are specific to that generator.
Tubemesh will supply distance along when it calls the segment generator which can use it to sample the varying properties.